### PR TITLE
docs(features): remove duplicate mention of dart

### PR DESCRIPTION
**Pull request by Xiphe**
in the "Flexible Development" section.

### CODE REVIEW

**Comment by Xiphe** [comment]
I signed it!

**Comment by Xiphe** [comment]
You're quite welcome :)

**Comment by PascalPrecht** [comment]
LGTM

File: public/features.jade [hunk 1]
```diff
@@ -54,7 +54,7 @@
     h3.text-headline.text-uppercase  Flexible Development
     p.text-body.
       The choice of language is up to you. In addition to providing full support for ES5 and
-      TypeScript, and Dart, Angular 2 works equally well with ES6, Dart, and other languages that
+      TypeScript, Angular 2 works equally well with ES6, Dart, and other languages that
```
Comment (kwalrath) [inline public/features.jade]: Good catch! I'd rather solve it the other way:

In addition to providing full support for ES5, TypeScript, and Dart, Angular 2 works equally well with ES6 and other languages that

**Comment by wardbell** [comment]
I'll just punch your change in, Kathy, and get this done.

**Comment by kwalrath** [comment]
Thanks, @wardbell.

**Pull request closed without merging**